debug/ftrace.rst

```diff
@@ -36,3 +36,42 @@ Usage
     - Run helper script called ``symbolize.py`` to translate the function graph
       addresses into function names: ``cat ftrace-<ta_uuid>.out |
       ./optee_os/scripts/symbolize.py -d <ta_uuid>.elf``
+
+Typical output
+**************
+
+.. code-block:: bash
+
+  # tracer: function_graph
+  #
+  # DURATION          FUNCTION CALLS
+  #  |   |             |   |   |   |
```

Comment:
This part is not present. Please copy real output (including the memory mappings which are necessary for `symbolize.py` to work properly.

Reply:
Yeah its not present but here my intention was to describe the columns with comments starting with `#`.

> Please copy real output (including the memory mappings which are necessary for symbolize.py to work properly).

Here reference output shown is after user has run `symbolize.py` script. I am not sure how user will benefit from function addresses and memory mappings that are essentially for `symbolize.py` script to operate on.

Reply:
> my intention was to describe the columns with comments 

Not really needed as it's quite obvious that column 1 is a duration and 2 is functions. Maybe this looks too similar to the Linux output, hence why I identify this as some generated text rather than a documentation comment. Fell free to do as you prefer (keep as is, rewrite differently or remove).

About the mappings, I have no strong opinion either. It is true that they are not related to the discussion. I was just hoping it would remind people they need to copy/paste the whole ftrace output into `symbolize.py`.

Reply:
> Not really needed as it's quite obvious that column 1 is a duration and 2 is functions.

Ok let me remove them instead.

> I was just hoping it would remind people they need to copy/paste the whole ftrace output into symbolize.py.

I think usage section is pretty clear on this.